CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Remove LegacyESVersion.V_7_6_ and V_7_7_ Constants ([#4837](https://github.com/opensearch-project/OpenSearch/pull/4837))
 - Remove LegacyESVersion.V_7_10_ Constants ([#5018](https://github.com/opensearch-project/OpenSearch/pull/5018))
 - Remove Version.V_1_ Constants ([#5021](https://github.com/opensearch-project/OpenSearch/pull/5021))
+- Removed unnecessary empty `build.gradle` files ([#4723](https://github.com/opensearch-project/OpenSearch/pull/4723))
```

Comment:
We can omit the changelog entry for build related changes like this. See https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#changelog

Reply:
ah, that rule changed in the meantime :)
i've removed it now (& also used the chance to rebase again, because why not)